woocommerce/woocommerce
# [ci] Only run lint jobs on pull_request event

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/47531

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Check the code changes.
Tested with:
- [this run](https://github.com/woocommerce/woocommerce/actions/runs/9173292069/job/25221678882?pr=47652), in which there was a .js file changed and the condition was `github.event_name == 'push'`, the job was skipped.
- [this run](https://github.com/woocommerce/woocommerce/actions/runs/9173337705/job/25222274155?pr=47652) in which there was a .js file changed and the condition was `github.event_name == 'pull_request'`, the job ran.
